doc/whats_new/v0.22.rst

```diff
@@ -714,6 +714,14 @@ Changelog
   - ``optimize.newton_cg`
   - ``random.random_choice_csc``
 
+:mod:`sklearn.voting`
+.....................
+
+- |Fix| The `named_estimators_` attribute in :class:`voting.VotingClassifier`
+  and :class:`voting.VotingRegressor` now correctly maps to dropped estimators.
+  Previously, `named_estimators_` mapped to estimators that were not dropped.
```

Comment:
Not sure that's correct either. As far as I understand this should be 

> Previously, the named_estimators_ mapping was incorrect whenever one of the estimators was dropped.

Reply:
Then name was wrong, the estimator was correct.

Reply:
It wasn't always incorrect. Maybe something like

> Previously, `named_estimators_` mapped to estimators that were not dropped
if the dropped estimator was ahead in the `estimators` list.

Reply:
The only case where `named_estimators_` wasn't wrong is when the dropped estimator was the last one. I wouldn't go into such details. We  can just say it was wrong.